README.md

```diff
@@ -18,6 +18,7 @@ We'd love to hear your feedback about `gh`. If you spot bugs or have features th
 - `gh pr [status, list, view, checkout, create]`
 - `gh issue [status, list, view, create]`
 - `gh repo [view, create, clone, fork]`
+- `gh gist [create, list, view, edit]`
```

Comment:
This is super minor nit and something that doesn't need to be addressed here, but it seems like these lists of available commands should be sorted somehow. Alphabetically maybe?

Reply:
we manually sort them in descending "importance" which is pretty subjective